### ISSUE TITLE
improvement: extend to support simulate multiple datapoint

### ISSUE THREAD

**Issue by henrywah196** [nenecmrf/OCPP-J-CP-Simulator#3]
Not sure if it is doable with current implementation. See if there's an efficient way to simulate multiple charge points

**Comment by kubarskii** [comment]
You may try to open several tabs in browser, it should work

**Comment by kubarskii** [comment]
> You may try to open several tabs in browser, it should work

Tried it. Works well.

**Comment by henrywah196** [comment]
yes. that's what I'm doing it right now. the problem is all the parameters seems hard coded. nice if you explore the way we can change it in different browser.

**Comment by kubarskii** [comment]
@henrywah196 

Okay i'll do it. Already started working on it.
